File: s3-inf-ppo/laboratoria/lab07/java/ppo7/src/ParkingLotManager/Entities/Car.java
package ParkingLotManager.Entities;

import ParkingLotManager.Interfaces.EntityInterface;

public class Car implements EntityInterface {

    private String plate;

    public Car(String plate) {
        this.plate = plate;
    }

    public String identify() {
        return "Car with plate number " + plate;
    }

    public boolean canEnter() {
        return true;
    }

}

File: s3-inf-ppo/laboratoria/lab07/php/src/Entities/Car.php
<?php

namespace ParkingLotManager\Entities;

use ParkingLotManager\Interfaces\EntityInterface;

class Car implements EntityInterface {

	protected $plate;

	public function __construct(string $plate) {
		$this->plate = $plate;
	}

	public function identify(): string {
		return "Car with plate number " . $this->plate;
	}

	public function canEnter(): bool {
		return true;
	}
	
}
